packages/opds-web-client/src/OPDSDataAdapter.ts

```diff
@@ -280,6 +281,24 @@ export function feedToCollection(feed: OPDSFeed, feedUrl: string): CollectionDat
     if (shelfLink) {
       shelfUrl = shelfLink.href;
     }
+
+    let aboutLabels = {
+      "about": "About",
+      "terms-of-service": "Terms of Service",
+      "privacy-policy": "Privacy Policy",
+      "copyright": "Copyright"
+    };
```

Comment:
circulation-patron-web feels right to me, since these are static-ish pages that exist for the client apps.